### PR TITLE
Add Game & GameManager classes

### PR DESCRIPTION
**Pull request by philmccarthy**
- add tests for Game & GameManager
- add Game class passing test
- add GameManager with `all` method to create game objects & add passing tests

### CODE REVIEW

File: lib/game_manager.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    CSV.foreach(@games_data_path, headers: true, header_converters: :symbol) do |row|
+      game_attributes = {
+                          game_id: row[:game_id],
+                          season: row[:season],
+                          away_team_id: row[:away_team_id],
+                          home_team_id: row[:home_team_id],
+                          away_goals: row[:away_goals],
+                          home_goals: row[:home_goals],
+                        }
+      @games << (Game.new(game_attributes))
```
Comment (elyhess) [inline lib/game_manager.rb:21]: no () needed

**Review by elyhess — CHANGES_REQUESTED** [review]
Please resubmit with the following changes.

**Comment by kaiheiongaku** [comment]
Looks good!

**Comment by kaiheiongaku** [comment]
Thank you for making the changes!

**Pull request merged**